extensions/asana/src/components/TaskActions.tsx

```diff
@@ -191,6 +191,12 @@ export default function TaskActions({ task, workspace, isDetail, mutateList, mut
           shortcut={{ modifiers: ["cmd"], key: "." }}
         />
 
+        <Action.CopyToClipboard
+          title="Copy Task Name and URL in Markdown Format"
```

Comment:
I'd go for something a bit shorter to avoid getting cropped text. What about something like `Copy Task URL as Markdown` or `Copy Task Formatted URL`?

Reply:
`Copy Task Formatted URL` is shorter. Let's go with that :)